README.md

```diff
@@ -1,8 +1,32 @@
-# Jina Hub
+<p align="center">
+<img src="https://github.com/jina-ai/jina-hub/blob/master/.github/Hub.png?raw=true" alt="Jina banner" width="200px">
```

Comment:
This isn't showing in the preview?

Reply:
yes, it doesn't show up - 
https://github.com/jina-ai/jina-hub/blob/hub-readme-update/README.md

Reply:
The banner will not show up in the preview because this `png` file is also being committed as a part of this PR, we are adding it to the `.github` path. It will show up on merging to master

Reply:
Great!